package.json

```diff
@@ -10,66 +10,66 @@
     ]
   },
   "dependencies": {
-    "@babel/core": "^7.18.6",
+    "@babel/core": "^7.19.0",
     "@centerforopenscience/markdown-it-imsize": "https://github.com/harigopal/markdown-it-imsize#esm",
-    "@emoji-mart/data": "^1.0.1",
+    "@emoji-mart/data": "^1.0.6",
     "@fortawesome/fontawesome-free": "^5.15.4",
     "@glennsl/bs-json": "^5.0.4",
     "@jeff-tian/markdown-it-katex": "^2.1.0",
     "@pnotify/mobile": "^5.2.0",
     "@prettier/plugin-ruby": "^1.6.1",
-    "@rails/ujs": "^6.1.6",
+    "@rails/ujs": "^6.1.7",
     "@reasonml-community/graphql-ppx": "^1.2.3",
     "@rescript/react": "^0.10.3",
     "animate.css": "3.7.2",
-    "autoprefixer": "^10.3.4",
+    "autoprefixer": "^10.4.9",
     "autosize": "^5.0.1",
     "babel-plugin-prismjs": "^2.1.0",
     "blob-polyfill": "^5.0.20210201",
     "bs-fetch": "^0.6.2",
     "bs-webapi": "^0.19.1",
```

Comment:
I think we should also think about updating `bs-webapi`
<img width="746" alt="image" src="https://user-images.githubusercontent.com/14979190/189766261-3940820a-107b-47ea-a5b7-dd410f7c780e.png">